_redirects.yml

```
@@ -557,6 +557,8 @@
 # feedback
 "https://docs.google.com/forms/d/e/1FAIpQLSenjK8KiBiOrwCpq06u_iVEZGv4tfTEpfmY_cQDmsqJrHL9Gw/viewform":
   - /feedback/extension/
+"https://docs.google.com/forms/u/1/d/e/1FAIpQLScgzKwqQyr7BmnphdrcGTO3wkye_McHJVsWzOIgEESfqmditQ/formResponse":
+  - /feedback/runtime
```

Will we use this for the containerd-integration only, or also beyond that? (thinking if this would be redirected elsewhere once that goes "GA", or if we intent to use this for other feedback after that)

Can you add a trailing slash to the URL?

```suggestion
  - /feedback/runtime/
```

---

Original thought was that we'd probably reuse this (even if we end up switching to a different form/survey platform).
Added the trailing `/`!